src/EA.Weee.Web/Views/Shared/_ViewTransferEvidenceNotePartial.cshtml

```diff
@@ -29,7 +38,8 @@
 </div>
 <div class="ui-helper-clearfix"></div>
 <h2 class="govuk-heading-m govuk-!-padding-top-5 govuk-!-margin-bottom-2">Total transferred (tonnes)</h2>
-<table id="transfer-note-tonnage-summary" class="govuk-table">
+
```

Comment:
Only display the page breaks in edit mode...

Reply:
oh okay